cluster.c

```diff
@@ -1132,6 +1146,7 @@ RRStatus computeHashSlot(RedisRaftCtx *rr, RaftReq *req)
             } else {
                 if (slot != thisslot) {
                     RedisModule_Free(keyindex);
+                    RedisModule_ReplyWithError(req->ctx, "CROSSSLOT Keys in request don't hash to the same slot");
```

Comment:
Looks like there is typo in CROSSSLOT but not sure what is the correct way. Btw should we start with ERR, something like `ERR cross slot keys ... `

Reply:
Seems like this is consistent with Redis, there's no typo but "cross" and "slot" combined:

```
src/cluster.c:        addReplyError(c,"-CROSSSLOT Keys in request don't hash to the same slot");
```

The first keyword is used as the error code. `-ERR` is the generic one, but certain errors have their own designated code which helps clients handle this / map it to individual exceptions.

Reply:
Ah right, thought there was one extra 'S' in the middle.